satpy/readers/seviri_l2_bufr.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017-2019 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""SEVIRI Bufr  format reader."""
+
+
+import logging
+from datetime import timedelta
+import numpy as np
+import xarray as xr
+import dask.array as da
+
+from satpy.resample import get_area_def
+
+import eccodes as ec
```

Comment:
eccodes needs to be mentionned in setup.py as an optional dependency at least

Reply:
done